docker-image-src/2024/coredb/docker-entrypoint.sh

```diff
@@ -159,7 +159,7 @@ function load_plugin_from_url
         echo >&2 "ERROR: could not query ${_versions_json_url} for plugin compatibility information.
     This could indicate a problem with your network or this container's network settings.
     Neo4j will continue to start, but \"${_plugin_name}\" will not be loaded."
-        return
+        return 1
```

Comment:
Is this going to break if we change `1` and `0` for `true` and `false`? Maybe not for this PR, but I just noticed

Reply:
I was just copying all the changes that'd happened from 5x onto calver, this was one of them. I think in bash it's considered normal to use `0` and `1` for true and false right?